CompilerSource/settings.h

```diff
@@ -43,7 +43,7 @@ namespace extensions
       windowLinks, graphicsLinks, audioLinks, collisionLinks, widgetLinks, networkLinks;
   };
   struct compiler_descriptor {
-    string identifier, resfile, buildext, buildname, runprog, runparam, builddir;
+    string target, identifier, resfile, buildext, buildname, runprog, runparam, builddir;
```

Comment:
Could you rename this to `target_compiler` or `compiler_descriptor_name` or something? It's not clear what it means, as it is.

Reply:
How about just "compiler" which I prefer because it's consistent with the other fields?
f63701c91ae096cfe26eb0d96a2666828cbadb70